### PR TITLE
make rmi messages more compatible with docker

### PR DESCRIPTION
**Pull request by baude**
in the case where we rmi an image that has only one reponame, we print
out an untagged reponame message.

$ sudo podman rmi busybox
Untagged: docker.io/library/busybox:latest
Deleted: db8ee88ad75f6bdc74663f4992a185e2722fa29573abcc1a19186cc5ec09dceb

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
Fixes: #2878

File: libpod/runtime_img.go [hunk 1]
```diff
@@ -93,7 +94,11 @@ func (r *Runtime) RemoveImage(ctx context.Context, img *image.Image, force bool)
 			err = errStorage
 		}
 	}
-	return img.ID(), err
+	if len(img.Names()) == 1 {
+		returnMessage = fmt.Sprintf("Untagged: %s\n", img.Names()[0])
```
Comment (mheon) [inline libpod/runtime_img.go]: Should we just `for` loop this?
```
returnMessage := ""
for _, name := img.Names() {
     returnMessage = returnMessage + fmt.Sprintf("Untagged: %s\n", name)
}
returnMessage = returnMessage + fmt.Sprintf("Deleted: %s", img.ID())
```

Comment (baude) [reply libpod/runtime_img.go]: No, but it does bring up an interesting edge case ... lemme go test

Comment (baude) [reply libpod/runtime_img.go]: yeah, added that change

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
Re-ran failed tests

**Comment by mheon** [comment]
Restoring lgtm
/lgtm
